test/camunda-cloud/integration/camunda-cloud-1-0.spec.js

```diff
@@ -0,0 +1,79 @@
+const { expect } = require('chai');
+
+const Linter = require('bpmnlint/lib/linter');
+
+const NodeResolver = require('bpmnlint/lib/resolver/node-resolver');
+
+const { readModdle } = require('../../helper');
+
+const config = {
+  extends: 'plugin:camunda-compat/camunda-cloud-1-0'
+};
+
+describe('integration - camunda-cloud-1-0', function() {
+
+  let linter;
+
+  beforeEach(function() {
+    linter = new Linter({
+      config,
+      resolver: new NodeResolver()
+    });
+  });
+
+
+  describe('valid', function() {
+
+    it('should be valid', async function() {
+
+      // given
+      const { root } = await readModdle('test/camunda-cloud/integration/camunda-cloud-1-0-valid.bpmn');
+
+      // when
+      const reports = await linter.lint(root);
+
+      // then
+      expect(reports).to.be.empty;
+    });
+
+  });
+
+
+  describe('invalid', function() {
+
+    it('should be invalid', async function() {
+
+      // given
+      const { root } = await readModdle('test/camunda-cloud/integration/camunda-cloud-1-0-invalid.bpmn');
+
+      // when
+      const reports = await linter.lint(root);
```

Comment:
Related to `allowed` vs. supported it would really help if we had at least a single test to verify the message. Do we have that?